core/models/user.py

```diff
@@ -106,6 +108,7 @@ class UserFeedback(models.Model):
                                    choices=option,
```

Comment:
With the `max_length` argument on the `report_type` field set at 10 characters, a user **cannot** submit feedback of type `feature_request` as it is 15 characters in length. Update that value a number greater than that.